recipes/glib/all/conanfile.py

```diff
@@ -82,7 +82,7 @@ def requirements(self):
             self.requires("libselinux/3.2")
         if self.settings.os != "Linux":
             # for Linux, gettext is provided by libc
-            self.requires("libgettext/0.20.1")
+            self.requires("gettext/0.20.1")
```

Comment:
```suggestion
            self.requires("libgettext/0.20.1")
```
Libgettext deprecation happened too soon, gettext is not yet able to replace it